webpack/components/extensions/HostDetails/Cards/HostCollectionsCard/HostCollectionsCard.js

```diff
@@ -101,14 +115,24 @@ const HostCollectionsDetails = ({
             }
           </Flex>
         </CardHeader>
-        <CardBody>
+        <CardBody className="host-collection-card-body">
+          {hostCollections?.length === 0 &&
+            <EmptyStateMessage
+              title={__('No host collections yet')}
+              body={__('To get started, add this host to a host collection.')}
+              customIcon={PlusCircleIcon}
+              showPrimaryAction
+              primaryActionButton={primaryActionButton}
+            />
+          }
           {hostCollections?.map((hostCollection) => {
             const {
               id, name, description, maxHosts, unlimitedHosts, totalHosts,
             } = propsToCamelCase(hostCollection);
             const isExpanded = expandedHostCollections.has(id);
             return (
               <Flex
+                style={{ marginTop: '50px' }}
```

Comment:
```suggestion
```

Reply:
It would look like this without this line.
![Screenshot from 2022-08-26 14-08-56](https://user-images.githubusercontent.com/2593270/186966274-727d9c80-1ce6-4b0b-b2a8-49cf5fe040e7.png)

Reply:
If you remove both this one and the one [up here](https://github.com/Katello/katello/pull/10239#discussion_r955412074), then it looks good. I think they cancel each other out 😄